src/Achievements/AchievementData.json

```
@@ -386,6 +386,11 @@
       "Name": "Speed demon.",
       "Description": "Destroy a bitnode in under 2 days."
     },
+    "BN_DESTROYER": {
+      "ID": "BN_DESTROYER",
+      "Name": "More BitNodes, please!",
+      "Description": "Destroy all BitNodes 3 times."
```

This is technically correct, but it still left me slightly confused the first time I read it.

Maybe
```suggestion
      "Description": "Reach SF x.3 in each BitNode."
```